### DISCUSSION TITLE
raspberry install

### DISCUSSION THREAD

**Discussion by martinwprior** [cagnulein/qdomyos-zwift#1174] [General]
after qmake qdomyos........
cannot find qdomyos-zwift_metatypes.json

Is this a problem?

**Comment by cagnulein** [comment]
nope, it's just a warning

**Reply by martinwprior** [reply]
when I run with -qml switch I still the get same panel as I did on my phone.
I also get error: XDG_RUNTIME_DIR not set in the environment
I have done a recompile to see whyif I can get the Raspberry to broadcast as FTMS

**Comment by cagnulein** [comment]
that error is ok.

**Reply by martinwprior** [reply]
Changed the Pi name to 'bike'. Now have domyosBridge in Zwift showing up

**Reply by cagnulein** [reply]
great!

**Comment by martinwprior** [comment]
My version for the Raspbery Pi is 2.12.45. it runs reliably. i have three
queries:

I do not get an email of my workout. Do I need to set up SMTP or something?
"Connect to Strava" just exits the setup page back to main page. I assumed
we would be passed to Strava or something like that.
Is there a procedure to apply updates or we should just rebuild?

Thank you

**Reply by cagnulein** [reply]
@martinwprior if you build from source you have to use your own credentials for smtp, strava and so on.

if you want i can build a version for raspberry with my credentials but i have also to add patreon license check (in order to pay the bills of email server and releated)

let me know if you are interested 

i was intending the raspberry version as a DYI version but of course it's doable both way

**Comment by martinwprior** [comment]
I do contribute through Patreon. There was hiccup last week as my card
expired and I had to re-enter my details in the email they sent me.

Well it would be nice when you have time. I will also look through the
code, which I have not done yet as I have been playing with setting it up
on a Raspberry 3 A+. I could not compile it just stopped the system. Maybe
overheated. Since put a fan and now trying DietPi OS. Its compiling as I
type. This time I am monitoring the process.

**Reply by cagnulein** [reply]
> I do contribute through Patreon. There was hiccup last week as my card
> expired and I had to re-enter my details in the email they sent me.
> 
> Well it would be nice when you have time. I will also look through the
> code, which I have not done yet as I have been playing with setting it up
> on a Raspberry 3 A+. I could not compile it just stopped the system. Maybe
> overheated. Since put a fan and now trying DietPi OS. Its compiling as I
> type. This time I am monitoring the process.
> 
> On Sun, Jan 22, 2023 at 4:04 PM Roberto Viola ***@***.***>
> wrote:
> 
> > @martinwprior <https://github.com/martinwprior> if you build from source
> > you have to use your own credentials for smtp, strava and so on.
> >
> > if you want i can build a version for raspberry with my credentials but i
> > have also to add patreon license check (in order to pay the bills of email
> > server and releated)
> >
> > let me know if you are interested
> >
> > i was intending the raspberry version as a DYI version but of course it's
> > doable both way
> >
> > —
> > Reply to this email directly, view it on GitHub
> > <https://github.com/cagnulein/qdomyos-zwift/discussions/1174#discussioncomment-4748277>,
> > or unsubscribe
> > <https://github.com/notifications/unsubscribe-auth/A4JUCAZMTET7HQRPOUESBATWTTSXNANCNFSM6AAAAAAT6WRGDI>
> > .
> > You are receiving this because you were mentioned.Message ID:
> > ***@***.***
> > com>
> >
> 

ok i will build ASAP

**Comment by martinwprior** [comment]
I keep forgetting On the build instructions it shows a linux gui for
qdmoyos-zwift (using --qml) I do not get it.

**Reply by cagnulein** [reply]
do you have a desktop environment on the raspberry?

**Comment by martinwprior** [comment]
Yes, I do.

**Reply by cagnulein** [reply]
so you need to install

apt install *qml*

**Comment by martinwprior** [comment]
First thing tomorrow morning :)

**Comment by martinwprior** [comment]
I have installed qml but still no change in interface. 
[debug-Thu_Jan_26_11_08_40_2023.log](https://github.com/cagnulein/qdomyos-zwift/files/10506104/debug-Thu_Jan_26_11_08_40_2023.log)

**Reply by cagnulein** [reply]
@martinwprior could you please attach a screenshot about what you're seeing? In the debug log i can see the QML interface. Thanks

**Comment by martinwprior** [comment]
IMG20230128082453.jpg
<https://drive.google.com/file/d/11P99TKZ8cTDn9m7lFrCAyOKwHg2Yq-cy/view?usp=drivesdk>

  IMG20230128082517.jpg
<https://drive.google.com/file/d/11JckOAyQ0burk6cYbt9QiBGfal7fZLeW/view?usp=drivesdk>
Sorry a bit late having small operation.

**Reply by cagnulein** [reply]
i asked permission to see the files

**Reply by cagnulein** [reply]
so @martinwprior now you got the QML interface. isn't it?

**Comment by martinwprior** [comment]
Sorry used to be able to just attach files. These days I really do not
understand. Maybe I will post on FB.

**Reply by cagnulein** [reply]
i got them  now, so you're getting the QML interface now, isn't it?
